core/src/test/resources/sql/functions.iq

```diff
@@ -499,6 +510,120 @@ select CONVERT(DATE, '05/01/2000', 103);
 !ok
 !}
 
+# [CALCITE-5771] Apply two different NULL semantics for CONCAT function(enabled in MySQL, Postgresql, BigQuery and MSSQL)
+select concat(null);
++--------+
+| EXPR$0 |
++--------+
+|        |
++--------+
+(1 row)
+
+!ok
+
+select concat(null, '');
++--------+
+| EXPR$0 |
++--------+
+|        |
++--------+
+(1 row)
+
+!ok
+
+select concat('', '');
++--------+
+| EXPR$0 |
++--------+
+|        |
```

Comment:
Stupid question: how can we distinguish if the expected result is the empty string (as in here) or null (as in the "concat in BigQuery" tests a few lines below)? Both cases look the same at first glance....

Reply:
As far as I know, we cannot distinguish them in functions.iq(null and "" behave the same since we don't use (null) for NULL result here). But we can tell the difference in SqlOperatorTest.

Reply:
What if we do something like:
```
select v, v is null from (select concat(null, '') as v);
```
On the "ambiguous" tests?

Reply:
Agreed. I add two test cases in functions.iq which help to tell the difference of the result is null or empty string.